### ISSUE TITLE
HomeAssistant not loading in VirtualBox

### ISSUE THREAD

**Issue by TwisterMc** [home-assistant/operating-system#2330]
### Describe the issue you are experiencing

I updated VirtualBox today and now HomeAssistant won't load. It's been working fine for months so I'm not sure what's going on. It's stuck on a screen that says "Booting Slot A" and there is nothing I can do.

<img width="1135" alt="image" src="https://user-images.githubusercontent.com/46997/215358287-2090dcd5-2572-4c29-9be4-90d802b5412f.png">

### What operating system image do you use?

ova (for Virtual Machines)

### What version of Home Assistant Operating System is installed?

Latest

### Did you upgrade the Operating System.

Yes

### Steps to reproduce the issue

1. Install latest version of VirtualBox on Mac.
2. Boot HomeAssistant

:shrug

### Anything in the Supervisor logs that might be useful for us?

```txt
00:00:03.250131 VMEmt: Halt method global1 (5) not available in driverless mode, using method1 (4) instead
00:00:03.250283 Changing the VM state from 'CREATING' to 'CREATED'
00:00:03.257737 Changing the VM state from 'CREATED' to 'POWERING_ON'
00:00:03.257960 Changing the VM state from 'POWERING_ON' to 'RUNNING'
00:00:03.257992 Console: Machine state changed to 'Running'
00:00:03.264664 EFI: debug point SEC_PREMEM
00:00:03.282592 GUI: UIMachineViewNormal::resendSizeHint: Restoring guest size-hint for screen 0 to 800x600
00:00:03.287819 GUI: HID LEDs sync is enabled
00:00:03.318540 GUI: UIMachineLogicNormal::sltCheckForRequestedVisualStateType: Requested-state=1, Machine-state=6
00:00:03.321464 ERROR [COM]: aRC=VBOX_E_VM_ERROR (0x80bb0003) aIID={4680b2de-8690-11e9-b83d-5719e53cf1de} aComponent={DisplayWrap} aText={Could not take a screenshot (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
00:00:03.409250 ERROR [COM]: aRC=VBOX_E_VM_ERROR (0x80bb0003) aIID={4680b2de-8690-11e9-b83d-5719e53cf1de} aComponent={DisplayWrap} aText={Could not take a screenshot (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
00:00:03.579241 EFI: VBoxDbg> loadimage64 'SecMain.efi' 0xfffcc094 LB 0x0
00:00:03.581599 EFI: VBoxDbg> loadimage64 'PeiCore.efi' 0x2020120 LB 0x0
00:00:03.584308 EFI: VBoxDbg> loadimage64 'PcdPeim.efi' 0x2026fe0 LB 0x2580
00:00:03.586565 EFI: VBoxDbg> loadimage64 'ReportStatusCodeRouterPei.efi' 0x2029620 LB 0xa00
00:00:03.587262 EFI: VBoxDbg> loadimage64 'StatusCodeHandlerPei.efi' 0x202a0a0 LB 0x700
00:00:03.587837 EFI: VBoxDbg> loadimage64 'PlatformPei.efi' 0x202a8a0 LB 0x5480
00:00:03.662326 EFI: debug point SEC_POSTMEM
00:00:03.663409 EFI: VBoxDbg> loadimage64 'PeiCore.efi' 0x7f364000 LB 0x6cc0
00:00:03.665953 EFI: VBoxDbg> loadimage64 'DxeIpl.efi' 0x7f361000 LB 0x2160
00:00:03.668119 EFI: VBoxDbg> loadimage64 'S3Resume2Pei.efi' 0x7f35f000 LB 0x1680
00:00:03.669141 EFI: VBoxDbg> loadimage64 'CpuMpPei.efi' 0x7f354000 LB 0xa8c0
00:00:03.713096 EFI: VBoxDbg> loadimage64 'TpmMmioSevDecryptPei.efi' 0x7f342000 LB 0xf00
00:00:03.713769 EFI: VBoxDbg> loadimage64 'Tcg2ConfigPei.efi' 0x7f340000 LB 0x1220
00:00:03.714641 EFI: VBoxDbg> loadimage64 'TcgPei.efi' 0x7f33d000 LB 0x26c0
00:00:03.715590 EFI: VBoxDbg> loadimage64 'Tcg2Pei.efi' 0x7f334000 LB 0x8520
00:00:03.717005 EFI: VBoxDbg> loadimage64 'DxeCore.efi' 0x7f318000 LB 0x1b100
00:00:03.720441 EFI: debug point DXE_CORE
00:00:03.723329 EFI: VBoxDbg> loadimage64 'DxeCore.efi' 0x7f318000 LB 0x0
00:00:03.747218 EFI: VBoxDbg> loadimage64 'DevicePathDxe.efi' 0x7e920000 LB 0x9e00
00:00:03.747857 EFI: VBoxDbg> loadimage64 'PcdDxe.efi' 0x7e92d000 LB 0x2f00
00:00:03.748623 EFI: VBoxDbg> loadimage64 'FvbServicesRuntimeDxe.efi' 0x7eeec000 LB 0x2100
00:00:03.752051 EFI: VBoxDbg> loadimage64 'ReportStatusCodeRouterRuntimeDxe.efi' 0x7eeea000 LB 0x1060
00:00:03.752774 EFI: VBoxDbg> loadimage64 'RuntimeDxe.efi' 0x7eee8000 LB 0x13c0
00:00:03.759055 EFI: VBoxDbg> loadimage64 'SecurityStubDxe.efi' 0x7e818000 LB 0x83940
00:00:03.759730 EFI: VBoxDbg> loadimage64 'EbcDxe.efi' 0x7e916000 LB 0x4120
00:00:03.760545 EFI: VBoxDbg> loadimage64 'Legacy8259.efi' 0x7e91d000 LB 0xb00
00:00:03.761804 EFI: VBoxDbg> loadimage64 'CpuIo2Dxe.efi' 0x7e91b000 LB 0xb00
00:00:03.762368 EFI: VBoxDbg> loadimage64 'CpuDxe.efi' 0x7e8fa000 LB 0xdd20
00:00:03.776988 EFI: debug point DXE_AP
00:00:03.779340 EFI: VBoxDbg> loadimage64 'IncompatiblePciDeviceSupportDxe.efi' 0x7e909000 LB 0x840
00:00:03.780036 EFI: VBoxDbg> loadimage64 'PciHotPlugInitDxe.efi' 0x7e8e2000 LB 0x2120
00:00:03.780817 EFI: VBoxDbg> loadimage64 'ResetSystemRuntimeDxe.efi' 0x7eee6000 LB 0x1040
00:00:03.781537 EFI: VBoxDbg> loadimage64 'Metronome.efi' 0x7e8e6000 LB 0x820
00:00:03.782232 EFI: VBoxDbg> loadimage64 'PrintDxe.efi' 0x7e8de000 LB 0x1f80
00:00:03.782905 EFI: VBoxDbg> loadimage64 'HiiDatabase.efi' 0x7e8a8000 LB 0x1a3a0
00:00:03.783621 EFI: VBoxDbg> loadimage64 'NullMemoryTestDxe.efi' 0x7e8e0000 LB 0x920
00:00:03.784303 EFI: VBoxDbg> loadimage64 'AcpiTableDxe.efi' 0x7e8d4000 LB 0x48c0
00:00:03.784962 EFI: VBoxDbg> loadimage64 'DpcDxe.efi' 0x7e8db000 LB 0xb20
00:00:03.785540 EFI: VBoxDbg> loadimage64 'IoMmuDxe.efi' 0x7e8d2000 LB 0x1780
00:00:03.786230 EFI: VBoxDbg> loadimage64 'EmuVariableFvbRuntimeDxe.efi' 0x7eee5000 LB 0xe40
00:00:03.786853 EFI: VBoxDbg> unload 'EmuVariableFvbRuntimeDxe.efi' # 0x7eee5000 LB 0xe40
00:00:03.793760 EFI: VBoxDbg> loadimage64 'VariableRuntimeDxe.efi' 0x7ee5c000 LB 0x89220
00:00:03.825962 EFI: VBoxDbg> loadimage64 'TcgDxe.efi' 0x7e8ca000 LB 0x3aa0
00:00:03.826900 EFI: VBoxDbg> unload 'TcgDxe.efi' # 0x7e8ca000 LB 0x3aa0
00:00:03.828184 EFI: VBoxDbg> loadimage64 'StatusCodeHandlerRuntimeDxe.efi' 0x7ee5b000 LB 0xa80
00:00:03.829070 EFI: VBoxDbg> loadimage64 'Timer.efi' 0x7e8d1000 LB 0x880
00:00:03.829269 PIT: mode=3 count=0x2e9c (11932) - 99.99 Hz (ch=0)
00:00:03.830342 EFI: VBoxDbg> loadimage64 'PciHostBridgeDxe.efi' 0x7e8c7000 LB 0x4c80
00:00:03.831811 EFI: VBoxDbg> loadimage64 'SetupBrowser.efi' 0x7e1a4000 LB 0x123c0
00:00:03.832443 EFI: VBoxDbg> loadimage64 'SmbiosDxe.efi' 0x7e8c4000 LB 0x2460
00:00:03.833265 EFI: VBoxDbg> loadimage64 'AcpiPlatform.efi' 0x7e8cd000 LB 0x720
00:00:03.834182 EFI: VBoxDbg> unload 'AcpiPlatform.efi' # 0x7e8cd000 LB 0x720
00:00:03.834920 EFI: VBoxDbg> loadimage64 'tftpDynamicCommand.efi' 0x7e40f000 LB 0x8800
00:00:03.835664 EFI: VBoxDbg> loadimage64 'httpDynamicCommand.efi' 0x7e199000 LB 0xaa60
00:00:03.836455 EFI: VBoxDbg> loadimage64 'LinuxInitrdDynamicShellCommand.efi' 0x7e408000 LB 0x6080
00:00:03.837102 EFI: VBoxDbg> loadimage64 'LogoDxe.efi' 0x7e185000 LB 0x9a80
00:00:03.837840 EFI: VBoxDbg> loadimage64 'FaultTolerantWriteDxe.efi' 0x7e405000 LB 0x2ca0
00:00:03.839609 EFI: VBoxDbg> loadimage64 'Tcg2Dxe.efi' 0x7e16d000 LB 0xbea0
00:00:03.840494 EFI: VBoxDbg> unload 'Tcg2Dxe.efi' # 0x7e16d000 LB 0xbea0
00:00:03.841377 EFI: VBoxDbg> loadimage64 'PcRtc.efi' 0x7ee58000 LB 0x23a0
00:00:03.848399 EFI: VBoxDbg> loadimage64 'SecureBootConfigDxe.efi' 0x7e0a7000 LB 0x6e120
00:00:03.849252 EFI: VBoxDbg> loadimage64 'WatchdogTimer.efi' 0x7e400000 LB 0x7e0
00:00:03.850065 EFI: VBoxDbg> loadimage64 'MonotonicCounterRuntimeDxe.efi' 0x7ee57000 LB 0x860
00:00:03.854236 EFI: VBoxDbg> loadimage64 'CapsuleRuntimeDxe.efi' 0x7ee56000 LB 0x920
00:00:03.855061 EFI: VBoxDbg> loadimage64 'DriverHealthManagerDxe.efi' 0x7e17b000 LB 0x41e0
00:00:03.855819 EFI: VBoxDbg> loadimage64 'BdsDxe.efi' 0x7e14d000 LB 0x16320
00:00:03.856645 EFI: VBoxDbg> loadimage64 'RamDiskDxe.efi' 0x7e169000 LB 0x81e0
00:00:03.857405 EFI: VBoxDbg> loadimage64 'DisplayEngine.efi' 0x7e12b000 LB 0x107a0
00:00:03.858831 EFI: VBoxDbg> loadimage64 'SmbiosPlatformDxe.efi' 0x7e167000 LB 0x1280
00:00:03.859727 EFI: VBoxDbg> loadimage64 'PlatformDxe.efi' 0x7e145000 LB 0x3620
00:00:03.860708 EFI: VBoxDbg> loadimage64 'PciBusDxe.efi' 0x7e09c000 LB 0xade0
00:00:03.861468 EFI: VBoxDbg> loadimage64 'VirtioPciDeviceDxe.efi' 0x7e14b000 LB 0x1120
00:00:03.862090 EFI: VBoxDbg> loadimage64 'Virtio10.efi' 0x7e142000 LB 0x2560
00:00:03.862704 EFI: VBoxDbg> loadimage64 'VirtioScsiDxe.efi' 0x7e149000 LB 0x1f20
00:00:03.863296 EFI: VBoxDbg> loadimage64 'PvScsiDxe.efi' 0x7e140000 LB 0x1a00
00:00:03.863888 EFI: VBoxDbg> loadimage64 'MptScsiDxe.efi' 0x7e13e000 LB 0x17e0
00:00:03.864572 EFI: VBoxDbg> loadimage64 'ConPlatformDxe.efi' 0x7e129000 LB 0x1c40
00:00:03.865275 EFI: VBoxDbg> loadimage64 'ConSplitterDxe.efi' 0x7e11d000 LB 0x5500
00:00:03.866057 EFI: VBoxDbg> loadimage64 'GraphicsConsoleDxe.efi' 0x7e119000 LB 0x3e80
00:00:03.866970 EFI: VBoxDbg> loadimage64 'TerminalDxe.efi' 0x7e092000 LB 0x4e80
00:00:03.867657 EFI: VBoxDbg> loadimage64 'QemuKernelLoaderFsDxe.efi' 0x7e09a000 LB 0x1900
00:00:03.868460 EFI: VBoxDbg> unload 'QemuKernelLoaderFsDxe.efi' # 0x7e09a000 LB 0x1900
00:00:03.869239 EFI: VBoxDbg> loadimage64 'DiskIoDxe.efi' 0x7e09a000 LB 0x1fa0
00:00:03.870128 EFI: VBoxDbg> loadimage64 'PartitionDxe.efi' 0x7e088000 LB 0x4f60
00:00:03.870827 EFI: VBoxDbg> loadimage64 'EnglishDxe.efi' 0x7e123000 LB 0xd80
00:00:03.871490 EFI: VBoxDbg> loadimage64 'ScsiBus.efi' 0x7e090000 LB 0x1e80
00:00:03.872552 EFI: VBoxDbg> loadimage64 'ScsiDisk.efi' 0x7e078000 LB 0x77c0
00:00:03.873201 EFI: VBoxDbg> loadimage64 'SataController.efi' 0x7e08e000 LB 0x1420
00:00:03.874067 EFI: VBoxDbg> loadimage64 'AtaAtapiPassThruDxe.efi' 0x7e070000 LB 0x70a0
00:00:03.874816 EFI: VBoxDbg> loadimage64 'AtaBusDxe.efi' 0x7e080000 LB 0x3720
00:00:03.875900 EFI: VBoxDbg> loadimage64 'NvmExpressDxe.efi' 0x7e062000 LB 0x6c20
00:00:03.876508 EFI: VBoxDbg> loadimage64 'SioBusDxe.efi' 0x7e086000 LB 0x1520
00:00:03.877308 EFI: VBoxDbg> loadimage64 'PciSioSerialDxe.efi' 0x7e05d000 LB 0x4b80
00:00:03.878079 EFI: VBoxDbg> loadimage64 'Ps2KeyboardDxe.efi' 0x7e059000 LB 0x33c0
00:00:03.878833 EFI: VBoxDbg> loadimage64 'VBoxVgaMiniPortDxe.efi' 0x7e06a000 LB 0x28a0
00:00:03.879627 EFI: VBoxDbg> loadimage64 'VBoxVgaDxe.efi' 0x7e051000 LB 0x32c0
00:00:03.880648 EFI: VBoxDbg> loadimage64 'VBoxHfs.efi' 0x7e043000 LB 0x61a0
00:00:03.881290 EFI: VBoxDbg> loadimage64 'VBoxSysTables.efi' 0x7e06f000 LB 0xa80
00:00:03.901933 EFI: VBoxDbg> loadimage64 'VBoxAppleSim.efi' 0x7e04e000 LB 0x2a40
00:00:03.903263 EFI: VBoxDbg> loadimage64 'VBoxApfsJmpStartDxe.efi' 0x7e057000 LB 0x1140
00:00:03.904093 EFI: VBoxDbg> loadimage64 'BootGraphicsResourceTableDxe.efi' 0x7e069000 LB 0xf00
00:00:03.905071 EFI: VBoxDbg> loadimage64 'Fat.efi' 0x7e035000 LB 0x6ec0
00:00:03.905721 EFI: VBoxDbg> loadimage64 'UdfDxe.efi' 0x7e03f000 LB 0x3dc0
00:00:03.906615 EFI: VBoxDbg> loadimage64 'VirtioFsDxe.efi' 0x7e029000 LB 0x5ac0
00:00:03.907358 EFI: VBoxDbg> loadimage64 'SnpDxe.efi' 0x7e023000 LB 0x5480
00:00:03.908153 EFI: VBoxDbg> loadimage64 'VlanConfigDxe.efi' 0x7e01d000 LB 0x5380
00:00:03.909068 EFI: VBoxDbg> loadimage64 'MnpDxe.efi' 0x7e00f000 LB 0x6d60
00:00:03.909675 EFI: VBoxDbg> loadimage64 'ArpDxe.efi' 0x7e031000 LB 0x3440
00:00:03.910676 EFI: VBoxDbg> loadimage64 'Dhcp4Dxe.efi' 0x7dfff000 LB 0x7fa0
00:00:03.912092 EFI: VBoxDbg> loadimage64 'Ip4Dxe.efi' 0x7dfdd000 LB 0x10400
00:00:03.912727 EFI: VBoxDbg> loadimage64 'Udp4Dxe.efi' 0x7dff7000 LB 0x7040
00:00:03.913874 EFI: VBoxDbg> loadimage64 'Mtftp4Dxe.efi' 0x7dfef000 LB 0x7640
00:00:03.915057 EFI: VBoxDbg> loadimage64 'TcpDxe.efi' 0x7dfc3000 LB 0xc4c0
00:00:03.916970 EFI: VBoxDbg> loadimage64 'UefiPxeBcDxe.efi' 0x7dfa3000 LB 0xf880
00:00:03.918959 EFI: VBoxDbg> loadimage64 'IScsiDxe.efi' 0x7df6d000 LB 0x1a1a0
00:00:03.922276 EFI: VBoxDbg> loadimage64 'VirtioNetDxe.efi' 0x7dfd9000 LB 0x3240
00:00:03.922921 EFI: VBoxDbg> loadimage64 'E1kNetDxe.efi' 0x7e007000 LB 0x2ce0
00:00:03.923588 EFI: VBoxDbg> loadimage64 'UhciDxe.efi' 0x7dfbe000 LB 0x4220
00:00:03.924260 EFI: VBoxDbg> loadimage64 'EhciDxe.efi' 0x7dfb9000 LB 0x4cc0
00:00:03.924933 EFI: VBoxDbg> loadimage64 'XhciDxe.efi' 0x7df9b000 LB 0x76e0
00:00:03.925622 EFI: VBoxDbg> loadimage64 'UsbBusDxe.efi' 0x7dfb4000 LB 0x4d40
00:00:03.926302 EFI: VBoxDbg> loadimage64 'UsbKbDxe.efi' 0x7dfd1000 LB 0x3e00
00:00:03.927058 EFI: VBoxDbg> loadimage64 'UsbMassStorageDxe.efi' 0x7df97000 LB 0x3100
00:00:04.368077 ERROR [COM]: aRC=VBOX_E_VM_ERROR (0x80bb0003) aIID={4680b2de-8690-11e9-b83d-5719e53cf1de} aComponent={DisplayWrap} aText={Could not take a screenshot (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
00:00:04.626700 Display::i_handleDisplayResize: uScreenId=0 pvVRAM=00007fcccb906000 w=1024 h=768 bpp=32 cbLine=0x1000 flags=0x0 origin=0,0
00:00:04.836072 PS2K: Selected scan set 2
00:00:04.935367 ATA: LUN#0: EXECUTE DEVICE DIAGNOSTIC, status 50
00:00:04.938787 ATA: LUN#0: INITIALIZE DEVICE PARAMETERS: 63 sectors per track, 16 heads
00:00:04.940575 ATA: LUN#2: EXECUTE DEVICE DIAGNOSTIC, status 00
00:00:04.964313 GUI: UIMediumEnumerator: Medium-enumeration finished!
00:00:05.384704 EFI: VBoxDbg> loadimage64 '.efi' 0x7d804000 LB 0x9a000
00:00:06.601149 EFI: debug point DXE_AP
```

### Anything in the Host logs that might be useful for us?

```txt
The only logs I know of are above.
```

**Comment by agners** [comment]
It seems you are using a rather old machine configuration with IDE disk drive emulation. Newer setups as per [installation guide](https://www.home-assistant.io/installation/windows/) should be using SCSI or SATA. Can you try a setup from scratch and compare your storage configuration to that of a new one?

**Comment by TwisterMc** [comment]
I ran though https://www.home-assistant.io/installation/macos/ and there's no change with the old image.

The new image gives me
<img width="979" alt="image" src="https://user-images.githubusercontent.com/46997/215658129-1cf3812d-6c82-4d02-92ab-22c859fe5576.png">

**Comment by asinla** [comment]
Did it get sorted? I was about to update my VirtualBox on my Mac.

**Comment by TwisterMc** [comment]
No fix yet. I did rollback VirtualBox with no success.

Turns out most of my stuff was TP Link / Kasa so I just bought [TPL Bridge](https://www.brooker.mobi/tpl-bridge/) and that works like a charm. I'll re-do home assistant some day.

**Comment by walshtj** [comment]
I had the same issue.  I updated Virtualbox from 6.1 to 7 and it 'broke' the VDI.
Long story short - I don't know what happened.  

I quickly gave up trying to fix it as I needed to get the house working again. So I downgraded to Virtualbox 6.1, reinstalled HA in a VM and used a (nightly) backup.

However, after that I started playing around again on a different (same spec) laptop. I created another HA instance and I've upgraded from VirtualBox from 6.1 to 7.  Then back to 6.1 then back to 7 - all without any problems. The reason I was doing this was to try to get Bluetooth to work with HA in a VM on a Mac (no success).

In summary, I think I was just unlucky the first time and something odd happened with the VDI which corrupted it

**Comment by asinla** [comment]
Yeah I've had mixed results too. 
I bought the Sonoff Zigbee 3.0 dongle and set everything up with Z2M eventually but it took very specific steps in VirtualBox for the dongle to show up. Once it did though it's been pretty stable. 
When you were trying to get Bluetooth working was it with a USB Bluetooth adapter or Mac built in? I believe there is no way to get Mac built in to work with VM's. I was going to try an external adapter.

My main issue right now is figuring out the networking stuff through Virtualbox. Can't cast anything to Google devices and my Sonos cannot reach it's port!


> I had the same issue. I updated Virtualbox from 6.1 to 7 and it 'broke' the VDI. Long story short - I don't know what happened.
> 
> I quickly gave up trying to fix it as I needed to get the house working again. So I downgraded to Virtualbox 6.1, reinstalled HA in a VM and used a (nightly) backup.
> 
> However, after that I started playing around again on a different (same spec) laptop. I created another HA instance and I've upgraded from VirtualBox from 6.1 to 7. Then back to 6.1 then back to 7 - all without any problems. The reason I was doing this was to try to get Bluetooth to work with HA in a VM on a Mac (no success).
> 
> In summary, I think I was just unlucky the first time and something odd happened with the VDI which corrupted it
> 
> EDIT: I find USB devices on Virtualbox on the Mac a total lottery in terms of getting them to work. On my test machine I've a spare Conbee II USB which VirtualBox 6.1 can find and use but Virtualbox 7 can't (some sharing violation). On my 'main' HA VM instance I don't use USB stick. Instead I have the original Raspberry Pi 3 running all the 'Deconz' stuff including the Conbee II stick. Ideally I'd like to integrate all the parts together on to a single old MacBook but its not easy.

**Comment by walshtj** [comment]
I bought one of those new SkyConnect USBs to test also (it's en route). However ultimately the issue is USB on VirtualBox on a Mac and I don't know if its Apple or Oracle's fault. If I ever got it all working I'd be afraid to update the Mac OS or the Virtualbox.

Regarding the Bluetooth I tried internal and a cheap external one (TFUFR Bluetooth 5.0 Dongle Adapter on Amazon.co.uk) - neither worked and after 2 days of following tutorials I just gave up again. If you get it working with a different Bluetooth adapter please tell us here.

**Comment by sk31337** [comment]
> I ran though https://www.home-assistant.io/installation/macos/ and there's no change with the old image.
> 
> The new image gives me <img alt="image" width="979" src="https://user-images.githubusercontent.com/46997/215658129-1cf3812d-6c82-4d02-92ab-22c859fe5576.png">

i have the exact same boot screen. Anyone knows how to fix this?

**Comment by thegrimester** [comment]
I also had this problem. I was installing HA on a N150 mini PC using VirtualBox running on Ubuntu. I followed the Linux instructions and downloaded the Linux vdi file. 

This did not work and the supervisor was crashing and I could not get to HA via port 8123.

The solution was to download the **Intel** vdi file from the windows installation instructions page. The files look identical and are the same size but there is obviously some difference for intel hardware vs Pi. 

if you are running in intel hardware give this a go.

Probably the instructions should be changed on HAs website
